### ISSUE TITLE
Performance on ScanNet in validation set

### ISSUE THREAD

**Issue by antao97** [hzykent/JSENet#3]
I use multiprocessing to accelerate the data preparation, and then run your code successfully.

But can you release your performance on ScanNet in validation set? I get 68.48 in validation set. I think this performance is comparable with KPConv, and thus I don't reproduce your performance.

**Comment by hzykent** [comment]
Since there are multiple loss terms for training, the weighting scheme is tedious to tune. As mentioned in the paper, the network is trained in two stage to bypass the weighting process. Please read the paper and follow the instructions.

**Comment by antao97** [comment]
I just run `python training_Scannet.py` following your instruction in README. Can you show me how to train network in two stage?

**Comment by hzykent** [comment]
> We first train our network without the joint refinement module for 350 epochs and then optimize the joint refinement module alone with the other parts fixed for 150 epochs.

The training process can be realized by changing the parameters updated by the optimizer. A well tuned weighting scheme is likely to further improve the performance by end-to-end training. You could try some if interested.
